packages/core/src/test.ts

```diff
@@ -1,39 +1,71 @@
-import Result, { Status } from './result';
-import Runnable from './runnable';
+import Result from './result';
+import Runnable, { isRunnable, RunnableOptions, RunnableTypes } from './runnable';
+import { RunOptions } from './runner';
+import Suite from './suite';
+
+export type testFn = () => void | Promise<any>;
```

Comment:
Just a style change to emphasize the correct type. Did I do this correctly :sweat_smile: ?
```suggestion
export type testFn = () => (void | Promise<any>);
```

Reply:
Also, type names should be capitalized: `TestFn`